### PR TITLE
[normal][30/5/2022]:Content

### PR DESCRIPTION
**Pull request by bisnhu-gurung123**
It contains PR for all related content

### CODE REVIEW

File: demo-app/src/app/app.component.scss [hunk 1]
```diff
@@ -1,17 +1,26 @@
 .container {
-  width: 1680px;
+  width: 1690px;
   margin: auto;
 }
 
 .card_A {
```
Comment (Sonam-Choden) [inline demo-app/src/app/app.component.scss:6]: class naming, BEM to be followed and no capital case

File: demo-app/src/app/app.component.scss [hunk 2]
```diff
@@ -1,17 +1,26 @@
...
   background-image: url("../assets/image/d1.jpeg");
```
Comment (Sonam-Choden) [inline demo-app/src/app/app.component.scss:7]: use background shorthand property

File: demo-app/src/app/app.component.scss [hunk 3]
```diff
@@ -1,17 +1,26 @@
...
   margin-top: 100px;
   background-repeat: no-repeat;
   background-size: cover;
 }
 
+.product {
+  width: 220px;
+  border-radius: 8px;
+  background-color: var(--higher);
+  color: #ffffff;
```
Comment (Sonam-Choden) [inline demo-app/src/app/app.component.scss:19]: use variables

File: demo-app/src/app/content/content.component.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<div class="card_A" fxLayout="row" fxLayoutAlign="space evenly center" fxLayoutGap="340px">
```
Comment (Sonam-Choden) [inline demo-app/src/app/content/content.component.html:1]: Proper class naming

File: demo-app/src/app/content/content.component.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
+<div class="content my-32 ">
```
Comment (bisnhu-gurung123) [inline demo-app/src/app/content/content.component.html:1]: content

File: demo-app/src/app/content/content.component.html [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  <div class="content__box" fxLayout="row" fxLayoutAlign="space evenly center" fxLayoutGap="340px">
```
Comment (bisnhu-gurung123) [inline demo-app/src/app/content/content.component.html:2]: demo_content

**Pull request merged**